include/mbedtls/platform.h

```diff
@@ -140,11 +143,11 @@ int mbedtls_platform_set_calloc_free( void * (*calloc_func)( size_t, size_t ),
 extern int (*mbedtls_fprintf)( FILE *stream, const char *format, ... );
 
 /**
- * \brief   Set your own fprintf function pointer
+ * \brief   This function allows configuring a custom \p fprintf function pointer.
  *
- * \param fprintf_func   the fprintf function implementation
+ * \param fprintf_func   The \c fprintf function implementation.
  *
- * \return              0
+ * \return               \c 0 on success.
```

Comment:
Blocker: this function always returns 0, so “0 on success” is semantic change. Please restore a simple “0” as the description of the return value. This goes for the other similar changes below.

Reply:
done